presentation/src/main/java/com/architect/coders/mu8/comics/ComicsActivity.kt

```diff
@@ -0,0 +1,82 @@
+package com.architect.coders.mu8.comics
+
+import android.os.Bundle
+import android.widget.ProgressBar
+import android.widget.TextView
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import androidx.lifecycle.Observer
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.architect.coders.mu8.R
+import com.architect.coders.mu8.data.remote.repository.ComicRepositoryImpl
+import com.architect.coders.mu8.comics.ComicsViewModel.UiModel
+import com.architect.coders.mu8.comics.ComicsViewModel.UiModel.*
+import com.architect.coders.mu8.utils.getViewModel
+import com.architect.coders.mu8.utils.makeItGone
+import com.architect.coders.mu8.utils.makeItVisible
+import com.architect.codes.mu8.interactor.comics.ComicsUseCaseImpl
+import com.google.android.material.snackbar.Snackbar
+
+class ComicsActivity : AppCompatActivity() {
+
+    private val progress: ProgressBar by lazy { findViewById<ProgressBar>(R.id.comic_activity_progress) }
+    private val recycler: RecyclerView by lazy { findViewById<RecyclerView>(R.id.comic_activity_recycler) }
+    private val GRID_COLUMS = 2
```

Comment:
This seems like a constant, please create a `private cons val` value instead.

Reply:
It is not possible at this level.

Reply:
I know is not, you have to declare a top level constant or inside a companion object. Also, please separate the views from the other variables with a new line.